### PR TITLE
Overflow set vertical

### PR DESCRIPTION
**Pull request by micahgodbolt**

### CODE REVIEW

**Comment by micahgodbolt** [comment]
image

**Comment by micahgodbolt** [comment]
Then replace the buttons with the new accordion I'm working on, and you've got a sidenav.

File: packages/office-ui-fabric-react/src/components/OverflowSet/OverflowSet.Props.ts [hunk 1]
```diff
@@ -27,6 +27,12 @@ export interface IOverflowSetProps extends React.Props<OverflowSet> {
   items?: IOverflowSetItemProps[];
 
   /**
+   * Direction that the items are layed out
+   * @default horizontal
+  */
+  itemDirection?: 'horizontal' | 'vertical';
+
```
Comment (dzearing) [inline packages/office-ui-fabric-react/src/components/OverflowSet/OverflowSet.Props.ts]: suggestion; make this `vertical?: boolean;`

**Comment by micahgodbolt** [comment]
overflowset

**Pull request merged**